src/routes/_surfaces/workspace/explore/TimeRangeSelector.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+  import Button from "$lib/components/Button.svelte";
+  import CaretDownIcon from "$lib/components/icons/CaretDownIcon.svelte";
+  import Menu from "$lib/components/menu/Menu.svelte";
+  import MenuItem from "$lib/components/menu/MenuItem.svelte";
+  import FloatingElement from "$lib/components/tooltip/FloatingElement.svelte";
+  import { onClickOutside } from "$lib/util/on-click-outside";
+  import { TimeRange, timeRanges } from "$lib/util/time-ranges";
+  import { tick } from "svelte";
+
+  const defaultTimeRange = timeRanges[0];
+  let selectedTimeRange: TimeRange = defaultTimeRange;
+
+  const prettyFormatTimeRange = (timeRange: TimeRange): string => {
```

Comment:
coudl we move these over to the utils directory for explore for now?

Reply:
Yep, good call